### PR TITLE
Change wording from avatar to profile picture

### PR DESCRIPTION
**Pull request by aaronraimist**
Fixes https://github.com/vector-im/element-meta/issues/1331 (assuming references to *room* avatars shouldn't be changed)

---
Here's what your changelog entry will look like:

## ✨ Features
 * Change wording from avatar to profile picture ([\#7015](https://github.com/matrix-org/matrix-react-sdk/pull/7015)). Fixes vector-im/element-meta#1331. Contributed by @aaronraimist.

Preview: https://61734318c0a89f4f04e4c61a--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Review by kittykat — APPROVED** [review]
Approving from the product team: we already use "profile picture" on iOS and Android. We should be consistent on Web.

**Comment by t3chguy** [comment]
Hey @aaronraimist any chance you could sort the merge conflicts?

File: src/components/views/avatars/BaseAvatar.tsx [hunk 1]
```diff
@@ -46,6 +46,8 @@ interface IProps {
     inputRef?: React.RefObject<HTMLImageElement & HTMLSpanElement>;
     className?: string;
     tabIndex?: number;
+    altText?: string;
+    ariaLabel?: string;
 }
 
 const calculateUrls = (url, urls, lowBandwidth) => {
```
Comment (t3chguy) [inline src/components/views/avatars/BaseAvatar.tsx:53]: Could these get types whilst you're here please

```suggestion
const calculateUrls = (url: string, urls: string[], lowBandwidth: boolean) => {
```

**Review by t3chguy — APPROVED** [review]
Looks sane otherwise!

**Comment by t3chguy** [comment]
Some test snapshots do need updating however

**Comment by kittykat** [comment]
Hey @aaronraimist , any chance you could follow up? Would be good to get this PR merged

**Comment by aaronraimist** [comment]
I'll try to get to this soon

**Pull request merged**